packages/desktop-client/src/components/tooltips.js

```diff
@@ -318,7 +319,7 @@ export class Tooltip extends Component {
           <div
             {...css(contentStyle, style, styles.darkScrollbar)}
             ref={this.contentRef}
-            data-testid="tooltip"
+            data-testid={this.props['data-testid'] || 'tooltip'}
```

Comment:
This allows us to write selectors for specific tooltips (i.e. datepicker or the filter menu).